packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -526,9 +528,14 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 
 	/** Package up the context's attach summary etc into an IAttachMessage */
 	private generateAttachMessage(localContext: IFluidDataStoreContextInternal): IAttachMessage {
-		const { attachSummary } = localContext.getAttachData(/* includeGCData: */ true);
+		// Get the attach summary.
+		const attachSummary = localContext.getAttachSummary();
 		const type = localContext.packagePath[localContext.packagePath.length - 1];
```

Comment:
oh funny, this code could have got it from the `getAttachData` call but didn't.

Having callers do that instead of returning it is good, it frees up this change.  I think I never considered that!